content/docs/next-release/sampling.md

```diff
@@ -12,18 +12,13 @@ When using configuration object to instantiate the tracer, the type of sampling
 * **Probabilistic** (`sampler.type=probabilistic`) sampler makes a random sampling decision with the probability of sampling equal to the value of `sampler.param` property. For example, with `sampler.param=0.1` approximately 1 in 10 traces will be sampled.
 * **Rate Limiting** (`sampler.type=ratelimiting`) sampler uses a leaky bucket rate limiter to ensure that traces are sampled with a certain constant rate. For example, when `sampler.param=2.0` it will sample requests with the rate of 2 traces per second.
-* **Remote** (`sampler.type=remote`, which is also the default) sampler consults Jaeger agent for the appropriate sampling strategy to use in the current service. This allows controlling the sampling strategies in the services from a [central configuration](#collector-sampling-configuration) in Jaeger backend, or even dynamically (see [Adaptive Sampling](https://github.com/jaegertracing/jaeger/issues/365)).
+* **Remote** (`sampler.type=remote`, which is also the default) sampler consults Jaeger agent for the appropriate sampling strategy to use in the current service. This allows controlling the sampling strategies in the services from a [central configuration](#collector-sampling-configuration) in Jaeger backend, or even dynamically (see [Adaptive Sampling](#adaptive-sampling)).
 
-### Adaptive Sampler
-
-Adaptive sampler is a composite sampler that combines two functions:
-
-  * It makes sampling decisions on a per-operation basis, i.e. based on {{< tip "span" >}} operation name. This is especially useful in the API services whose endpoints may have very different traffic volumes and using a single probabilistic sampler for the whole service might starve (never sample) some of the low QPS endpoints.
-  * It supports a minimum guaranteed rate of sampling, such as always allowing up to N {{< tip "traces" "trace" >}} per seconds and then sampling anything above that with a certain probability (everything is per-operation, not per-service).
+## Collector Sampling Configuration
 
-Per-operation parameters can be configured statically or pulled periodically from Jaeger backend with the help of Remote sampler. Adaptive sampler is designed to work with the upcoming [Adaptive Sampling](https://github.com/jaegertracing/jaeger/issues/365) feature of the Jaeger backend.
+If your clients are configured to use remote sampling then sampling rates can be centrally controlled via the collectors. In a remote sampling setup a json document is served to the Jaeger client that describes endpoints and their sampling probabilities. This document can be generated two different ways: [statically](#static-sampling) or [dynamically](#adaptive-sampling). The method of document generation is controlled by the environment variable `SAMPLING_TYPE` which can be set to either `static`(default) or `adaptive`.
```

Comment:
> `[statically](#static-sampling)`

just thinking out loud - the name "static" is actually a misnomer. It used to be accurate but now this same file can be reloaded on changes from the file system, so "static" sounds misleading. We may want to rename the value to `file` instead of `static`, and change the docs here accordingly to not even mention `static`.

> environment variable `SAMPLING_TYPE`

I may have missed it, but this var doesn't seem to get printed in `env` command
```shell
$ SPAN_STORAGE_TYPE=memory go run ./cmd/collector env
...
```
Since we didn't do a release with it, I would strongly suggest renaming SAMPLING_TYPE -> SAMPLING_CONFIG_TYPE